client/components/advanced-credentials/credentials-form/index.tsx

```diff
@@ -455,7 +455,7 @@ const ServerCredentialsForm: FunctionComponent< Props > = ( {
 						placeholder={ translate( 'example.com' ) }
 						value={ formState.host }
 						onChange={ handleFormChange }
-						disabled={ disabled }
+						disabled={ true }
```

Comment:
Hi there, this is causing the credential form to break for Jetpack backup, see https://github.com/Automattic/jpop-issues/issues/8842 along with some UI style issues since it is a shared component.

Reply:
Hey @rustamshaikh, I'm reverting this here https://github.com/Automattic/wp-calypso/pull/87640. When @agrullon95 is online we can put shape a better PR

Reply:
@rustamshaikh revert has been deployed ✌️